src/actions/measurements.js

```diff
@@ -170,7 +170,8 @@ const getCollectionDisplayControls = (controls, collection) => {
 };
 
 const parseMeasurementsJSON = (json) => {
-  const collections = json["collections"];
+  // Avoid editing the original json values since they are cached for narratives
+  const collections = cloneDeep(json["collections"]);
```

Comment:
[aside]

When we're not viewing a narrative we could avoid this and reduce the memory footprint. I don't actually want to do it because it makes testing harder and will invariably result in bugs somewhere, but something that came to mind as I read this.

Reply:
Yeah, I thought about adding some conditionals around narratives, but like you said, it seemed like that would eventually result in bugs somewhere 🙈